### ISSUE TITLE
In full-screen mode, Cmd+Tab immediately quits emulation

### ISSUE THREAD

**Issue by matthewjamesadam** [OpenEmu/OpenEmu#4387]
If you are running a game in full-screen mode and hit Cmd+Tab, emulation quits immediately.  This does not happen in windowed mode.  I've reproduced this on both Nestopia and Snes9x cores.

# Steps to reproduce

1. Run a game
1. Switch to full-screen mode
1. Hit Cmd+Tab to switch to some other window
1. Hit Cmd+Tab to switch back

# Expected Behavior

I'd expect emulation to pause while a different window has focus, as it does in windowed mode.

# Observed Behavior

The OpenEmu game window disappears.  OpenEmu is still running and the OpenEmu library window is still there.  If you re-start emulation for the game, you do not start from the point that you were last at, you lose any progress you've made in the game.

# Debugging Information

- OpenEmu Version: 2.3
- macOS Version: 11.1

Note: I did not see this in the previous versions of OpenEmu, so I suspect this is a new regression.

**Comment by clobber** [comment]
I tried this out and it seems the window just gets hidden by doing cmd+tab. I checked the "Window" menu item under OpenEmu and saw an entry for my running game and was able to click to restore. Can you replicate this?

**Comment by matthewjamesadam** [comment]
You're right!  I had an awful lot of hidden windows there.  I think this is pretty confusing behaviour, I'd suggest that it might be less surprising if the window were moved out of full-screen when this happens (I think this is what previous versions of OpenEmu did?)

**Comment by clobber** [comment]
The previous release did not default to the popout window, so offhand I don't know about the popout, but yes I agree this is unexpected UX and should be fixed.
